vcxagency-node/src/configuration/app-config.js

```diff
@@ -65,8 +70,19 @@ const configValidation = Joi.object().keys({
   PG_WALLET_CONNECTION_TIMEOUT_MINS: Joi.number().integer().min(1).max(100)
 })
 
+function testConfigPathExist (appConfig, key) {
+  const path = appConfig[key]
+  if (!fs.existsSync(path)) throw Error(`${key} = ${path} is not a valid path or the path does not exist`)
+}
+
 function validateAppConfig (appConfig, callback) {
   Joi.validate(appConfig, configValidation, callback)
+  if (appConfig.SERVER_ENABLE_TLS === 'true') {
+    if (!appConfig.CERTIFICATE_PATH || !appConfig.CERTIFICATE_KEY_PATH) throw Error('Valid certificate and key paths must be specified when TLS enabled!')
```

Comment:
I'd prefer to use `{}` for all if statements, without is more error prone

Reply:
I would disagree as long as the statement is on the same line as condition, and I would also say the result is easier to read, since it reduces the number of lines.